tests/test_config.py

```diff
@@ -25,38 +25,32 @@ class TestConfig(TestCase):
     def setUp(self):
         """Set up. Creates a fake config"""
         self.config_test = _Config()
+        if os.path.exists(config.FILEPATH):
+            os.remove(config.FILEPATH)
```

Comment:
Don't do this kind of action on "config.FILEPATH", it looks like you're deleting the real file.

Bring the change you're doing globally below (as I said in the last review), do that here, and operate on the temp file.

Example (untested):
```
    temp_file = "/tmp/blah"
    config.FILEPATH = temp_file  # but you need to restore it to the real value, as part of the cleanup, see mock.patch()
    
    os.remove(temp_file)  # later in tearDown
```